### ISSUE TITLE
Change contrast medium value to be a11y conform

### ISSUE THREAD

**Issue by muellernaumann** [porsche-design-system/porsche-design-system#2394]
Scope
---
Replace the current value of our Contrast Medium (light) with `#6B6D70` • `filter: invert(45%) sepia(6%) saturate(235%) hue-rotate(177deg) brightness(91%) contrast(84%);` and Contrast Medium (dark) with `#88898C` • `filter: invert(66%) sepia(3%) saturate(333%) hue-rotate(187deg) brightness(82%) contrast(86%);` as it does not meet the contrast ratio of 4.5:1 (AA).

Acceptance criteria
---
- [x] Contrast-Medium (light) has the value `#6B6D70` • `filter: invert(45%) sepia(6%) saturate(235%) hue-rotate(177deg) brightness(91%) contrast(84%);`
- [x] Contrast-Medium (dark) has the value `#88898C` • `filter: invert(66%) sepia(3%) saturate(333%) hue-rotate(187deg) brightness(82%) contrast(86%);`
- [x] Contrast-Medium (light) and Contrast-Medium (dark) has a contrast ratio of min 4.5:1 (AA) on `background-base` and `background-surface`
- [x] Also optimized color values from #2408 must be integrated
- [x] Crosscheck on mobile safari 
- [x] `color-contrast` is re-enabled in A11y E2E test for Storefront (`packages/storefront/tests/e2e/helpers/axe-helper.ts`)
